v2/robotmk/src/bin/scheduler/scheduling/scheduler.rs

```diff
@@ -47,7 +58,7 @@ async fn run_suite_scheduler(suite: Suite) {
 }
 
 async fn run_cleanup_job(cancellation_token: CancellationToken, suites: Vec<Suite>) {
-    let mut clock = interval(Duration::from_secs(300));
+    let mut clock = interval_at(compute_start_time(300), Duration::from_secs(300));
```

Comment:
It seems to me that calling `tick().await` on `clock` has a very similiar effect.

Reply:
Not sure what you mean. If we dont use `interval_at` but `interval`, the first tick will complete immediately.